tests/integration/mixer/telemetry/metrics/blackhole/blackhole_test.go

```diff
@@ -0,0 +1,117 @@
+//  Copyright 2020 Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package blackhole
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/environment"
+	"istio.io/istio/pkg/test/framework/components/galley"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/components/mixer"
+	"istio.io/istio/pkg/test/framework/components/namespace"
+	"istio.io/istio/pkg/test/framework/components/prometheus"
+	"istio.io/istio/pkg/test/framework/components/sleep"
+	"istio.io/istio/pkg/test/framework/label"
+	"istio.io/istio/pkg/test/framework/resource"
+	util "istio.io/istio/tests/integration/mixer"
+)
+
+var (
+	sleepNs    namespace.Instance
+	ist        istio.Instance
+	prom       prometheus.Instance
+)
+
+func TestBlackHoleClusterMetric(t *testing.T) {
```

Comment:
It would be good to split this into tcp and http metric tests .

Reply:
Agreed. I'll also add some documentation on how it was setup as it's not clear that a wildcard virtual outbound listener on port 80 is being used (and not in the case of 443).

Reply:
Ended up splitting them into network response / telemetry tests. See if you like the changes before I resolve this.